### PR TITLE
Add Fabric/Crashlytics.

### PR DESCRIPTION
**Pull request by judytuna**

### CODE REVIEW

File: ios/Hubbub/LoginViewController.swift [hunk 1]
```diff
@@ -9,6 +9,8 @@
 import Firebase
 import p2_OAuth2
 import UIKit
+import Crashlytics
```
Comment (justinrosenthal) [inline ios/Hubbub/LoginViewController.swift]: Do we actually want to commit the stuff in this file, or was it just for debugging/verifying the integration?

Comment (judytuna) [inline ios/Hubbub/LoginViewController.swift]: Good point. Let's delete the crash button, but lines 81-83 add a "log in" Event to Answers on Fabric. If we like that, then we'd keep it for Answers.logLogin... What do you think?

**Comment by judytuna** [comment]
@justinrosenthal I rebased master and hopefully fixed all the merge commits. Can you double-check that things look okay?

**Comment by justinrosenthal** [comment]
:shipit:

**Pull request merged**